app/services/mapping_migration_service.rb

```diff
@@ -0,0 +1,90 @@
+class MappingMigrationService
+  attr_reader :integration_name, :rtp_id, :template_file, :templates_dir, :upload_integrations
+
+  def initialize
+    @templates_dir = Configuration.paths_templates_plugins
+    @upload_integrations = Dradis::Plugins::with_feature(:upload)
```

Comment:
Do we need `upload_integrations` as an attribute considering we're only using it one spot?